src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,113 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String createQuery = "INSERT INTO manufacturers(name, country) values (?, ?);";

```

Comment:
All SQL keywords should be uppercase
![image](https://user-images.githubusercontent.com/32339836/227139565-7f30f13d-9960-4c0d-8b6c-793d97c68e3f.png)
```suggestion
        String createQuery = "INSERT INTO manufacturers(name, country) VALUES (?, ?);";
```